src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -458,17 +466,18 @@ object GermlineAssemblyCaller {
 
           possibleVariant
         })
-
       })
+
     variants.toSet
   }
 
   /**
-    * Call variants using a pileup and genotype likelihoods
-    * @param pileup Pileup at a given locus
-    * @return Possible set of called variants
-    */
-  def callPileupVariant(pileup: Pileup): Set[CalledAllele] = {
+   * Call variants using a pileup and genotype likelihoods
```

Comment:
yea, per [my comment](https://github.com/hammerlab/guacamole/pull/437#issuecomment-208975109) on #437, i think the defaults for scalariform and IJ conflicted, but IJ can go either way (maybe scalariform can too?).

I realigned them here partly because the method above had them this way.

I also added the blank line, which I thought was standard.